cmd/agent/main.go

```diff
@@ -45,7 +45,10 @@ func main() {
 	flags.AddFlagSet(o.Flags())
 	local := flag.NewFlagSet(os.Args[0], flag.ExitOnError)
 	klog.InitFlags(local)
-	local.Set("v", "4")
+	err := local.Set("v", "4")
+	if err != nil {
+		klog.Errorf("error: %v\n", err)
```

Comment:
There is a certain optimism inherent in attempting to use klog when you just received an error when configuring klog. Not saying I have a better idea but I wonder if it might be worth also sending the error message to stderr.

Reply:
Also please use structured logging in you PR. 
Eg. 
klog.ErrorS(err, "Failed to set log level")

Reply:
thanks, let's try fmt.Fprint in those cases when klog initialization happens